# Generated from: /home/latty/link_dir/2015_li3ds/__ROS__/Ellipse_N/ROS/overlay_ws/src/sbg_ros_driver/sbgECom/src/binaryLogs/sbgEComBinaryLogGps.h
# SbgLogGpsHdt
uint32 timeStamp
uint16 status
uint32 timeOfWeek
float32 heading
float32 headingAccuracy
float32 pitch
float32 pitchAccuracy